microsoft-azure-sb/3.0/modules/ROOT/pages/index.adoc

```diff
@@ -10,7 +10,7 @@ This connector supports integrations with Azure Service Bus (cloud). It does not
 
 To use the Azure management API for CRUD operations of queues, topics, subscriptions, and rules, use Anypoint Connector for Azure Service Bus Management (Azure Service Bus Management Connector).
 
-For compatibility information and fixed issues, see the xref:release-notes::connector/<connector>.adoc[Azure Service Bus Messaging Connector Release Notes] 
+For compatibility information and fixed issues, see the xref:release-notes::connector/<connector>.adoc[Azure Service Bus Connector Release Notes] 
```

Comment:
Does this link have to be here or can we add a ==See Also topic?
Per our new linking guidelines, we want to avoid "nice to have" links directly in narrative.